app/views/mailers/notifications_mailer/acquired_trophy.html.haml

```diff
@@ -0,0 +1,6 @@
+%p Hi #{@user.handle},
+%p
+  Congratulations! You've unlocked a new trophy at Exercism.
```

Comment:
```suggestion
  Congratulations! You've unlocked a new trophy on the #{track.name} Track.
```

Reply:
(I don't know if we have `track` as a variable or need to do `trophy.track` etc.